css/media.css

```diff
@@ -395,3 +396,537 @@
+    padding-bottom: 52px;
+  }
+
+  .aside__text {
+    margin-top: 25px;
+    padding-right: 43px;
+  }
+
+  .wrapp__aside-content {
+    width: 41%;
+    margin-top: 12px;
+  }
+
+  .aside__display {
+    margin-bottom: 28px;
+  }
+
+  .aside__volum {
+    margin-bottom: 70px;
+  }
+
+  .aside__footer a {
+    font-size: 16px;
+    line-height: 20px;
+  }
+
+  .aside__footer-img {
+    width: 33px;
+    height: 33px;
+  }
+
+  .aside__footer-button {
+    width: 221px;
+    padding-top: 15px;
+    height: 55px;
+  }
+
+  .aside__footer-btn {
+    margin-right: 40.5px;
+  }
+
+  .aside__footer {
+    padding-right: 0;
+  }
+}
+
+@media (max-width: 560px) {
+  .wrapp__aside-content {
+    margin-right: 10px;
+  }
+}
+
+@media (max-width: 536px) {
+  .wrapp__aside-content {
+    display: none;
+  }
+
+  .aside__footer {
+    display: none;
+  }
+
+  .wrapp__aside-title {
+    margin-left: 30px;
+    width: 100%;
+  }
+
+  .aside {
+    padding-bottom: 0;
+  }
+}
+
+@media (max-width: 444px) {
+  .main__title {
+    display: none;
+  }
+
+  .main-bg {
+    height: 337px;
+  }
+}
+
+@media (max-width: 375px) {
+  .main-bg {
+    background-image: url(../IMG/little-bg.png);
+    background-size: 100%;
+    background-repeat: no-repeat;
+    height: 329px;
+  }
+
+  .aside {
+    padding-top: 18px;
+  }
+
+  .aside__text {
+    margin-top: 12px;
+    margin-bottom: 34px;
+  }
+}
+
+@media (max-width: 1424px) {
+  .aside__footer-btn {
+    margin-right: 14px;
+  }
+}
+
+@media (max-width: 1400px) {
+  .aside__footer-btn {
+    margin-right: 10px;
+  }
+}
+
+@media (max-width: 1376px) {
+  .inner-aside {
+    padding: 51px 40px 60px 40px;
+  }
+}
+
+@media (max-width: 1365px) {
+  .stars {
+    padding: 0;
+  }
+
+  .aside__footer-btn {
+  }
+
+  .wrapper-stars {
+    padding: 0 15px 0 15px;
+  }
+
+  .stars__img {
+    margin-left: 6%;
+  }
+}
+
+@media (max-width: 1320px) {
+  .star-group {
+    margin-left: 5px;
+  }
+}
+
+@media (max-width: 1276px) {
+  .inner-aside {
+    padding: 51px 20px 60px 20px;
+  }
+
+  .star {
+    padding-left: 1px;
+  }
+}
+
+@media (max-width: 1200px) {
+  .star {
+    padding-right: 1px;
+  }
+
+  .section__aside-title {
+    font-size: 25px;
+    text-align: left;
+  }
+
+  .bg-block {
+    height: 126px;
+  }
+
+  .content__title {
+    font-size: 25px;
+  }
+
+  .section__aside-text {
+    padding: 0;
+  }
+
+  .section__inner-info {
+    padding: 0 8% 6% 0;
+  }
+
+  .srars-info {
+    margin-bottom: 13%;
+  }
+
+  .aside-stars__footer {
+    padding-right: 0;
+  }
+}
+
+@media (max-width: 1144px) {
+  .wrapper-stars {
+    padding: 0;
+  }
+}
+
+
+@media (max-width: 1076px) {
+  .stars__img {
+    margin-left: 0;
+  }
+
+  .bg-block {
+    width: 92%;
+    right: 10px;
+  }
+
+  .inner-aside {
+    padding: 23px 20px 23px 20px;
+  }
+
+  .block {
+    left: -17px;
+  }
+
+  .logo-school {
+    right: 13px;
+    bottom: -32%;
+  }
+
+  .star__item {
+    font-size: 14px;
+  }
+
+.star__item span {
+  left: 53px;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5948685)